job-server/config/local.conf.template

```diff
@@ -39,8 +39,9 @@ spark {
     # in case spark distribution should be accessed from HDFS (as opposed to being installed on every mesos slave)
     # spark.executor.uri = "hdfs://namenode:8020/apps/spark/spark.tgz"
 
-    # uris of jars to be loaded into the classpath for this context
+    # uris of jars to be loaded into the classpath for this context. Uris is a string list, or a string separated by commas ','
     # dependent-jar-uris = ["file:///some/path/present/in/each/mesos/slave/somepackage.jar"]
+    # dependent-jar-uris = "file:///some/path/of/my-foo-lib.jar,file:///some/path/of/my-boo-lib.jar"
```

Comment:
I'd prefer to keep the example as a string list.  Much more type or parser safe.